qoqo_qasm/qoqo_qasm/__init__.py

```diff
@@ -23,10 +23,11 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 from qoqo_qasm.__version__ import __version__
```

Comment:
Since we are switching over to a uniform rust backend we can also remove the __version__